met-api/src/met_api/resources/submission.py

```diff
@@ -135,11 +135,20 @@ def get(survey_id):
                 sort_key=args.get('sort_key', 'submission.id', str),
                 sort_order=args.get('sort_order', 'asc', str),
             )
+            advanced_search_filters = {
+                'status': args.get('status', None, int),
+                'comment_date_to': args.get('comment_date_to', None, str),
+                'comment_date_from': args.get('comment_date_from', None, str),
+                'reviewer': args.get('reviewer', None, str),
+                'reviewed_date_from': args.get('reviewed_date_from', None, str),
+                'reviewed_date_to': args.get('reviewed_date_to', None, str),
+            }
             submission_page = SubmissionService()\
                 .get_paginated(
                     survey_id,
                     pagination_options,
                     args.get('search_text', '', str),
+                    advanced_search_filters if any(advanced_search_filters.values()) else None
```

Comment:
`advanced_search_filters if any(advanced_search_filters.values()) else None`
may be handle this if check inside the submissionService().get_paginated() method so that incase if we have more methods calling get_paginated,each method dont have to perform the same check